CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## x.x.x
+- Fixed sending '--' as carrier name value due to deprecated calls from iOS version 16.4
+
```

Comment:
how about?
- Dropped support for carrier information tracking for iOS 16.4 and  up (due to iOS API changes)